controllers/territory_controller/territory_controller.py

```diff
@@ -106,33 +107,45 @@ class TerritoryRoot(str, enum.Enum):
 }
 
 
-ClaimLogEntry = Tuple[StationCode, Claimant, float, bool]
+@dataclass
+class StationStatus:
+    owner: Claimant = Claimant.UNCLAIMED
+    locked: bool = False
+
```

Comment:
Using both a `dataclass` and `NamedTuple` here concerns me, but I'm not sure why. Is there a reason it's a `dataclass`?

Reply:
Presumably because it needs to be mutable?

If anything I'd perhaps suggest making the other also be a `dataclass`, though maybe adding `frozen=True` if we want it immutable?

Reply:
51e27b73624e907c64e5943f74a1e10dbb089397